src/library/scala/Array.scala

```diff
@@ -124,16 +124,16 @@ object Array {
     * @see `java.util.Arrays#copyOf`
     */
   def copyOf[A](original: Array[A], newLength: Int): Array[A] = ((original: @unchecked) match {
-    case x: Array[BoxedUnit]  => newUnitArray(newLength).asInstanceOf[Array[A]]
```

Comment:
unused, so naming it `original` invokes the shadowing rule. That is, the shadowing name does not introduce new information (except the narrower type). Then it's nicer to use the same name for all cases.